### ISSUE TITLE
sh: : command not found

### ISSUE THREAD

**Issue by geek876** [mikehaertl/phpwkhtmltopdf#243]
Hi,
We using 
```
wkhtmltopdf 0.12.4 (with patched qt)
```
OS: Centos6.6 , Nginx , PHP-FPM.

It fails on line 302 of WkHtmlToPdf.php
```
$stderr = stream_get_contents($pipes[2]);
```
with $stderr: "sh: : command not found\n" and hence the pdf file is got getting created.

The `$command` variable does get constructed as
```
'' --no-outline --encoding 'UTF-8' --footer-right '[page]/[toPage]' --margin-top '10' --margin-left '10' 'http://xxx/xxx/xxx/pdf_files/Special Privileges.html' '/tmp/tmp_WkHtmlToPdf_g5zHud'
```
So not sure what is the issue. ?

This was working perfectly fine when we had 'apache'. Now we moved to 'nginx, php-fpm' and since then it has stopped working. Nothing was changed at 'wkhtmltopdf' level, i.e. path, version etc is all same (untouched) and so is code. Just changed apache to 'nginx php-fpm'.

wkhtmltopdf is working fine at cli level and is in path and all file perms are good as far as i can see.

When we run with apaache, the $stderr does'nt throw any errors and pdf generation is fine.

**Comment by mikehaertl** [comment]
Seems like `wkhtmltopdf` is not found anymore. You try to configure the full path to `wkhtmltopdf` in the `binary` [option](https://github.com/mikehaertl/phpwkhtmltopdf#wrapper-options). It could also have to do with changes apparmor on that system. If that's the case I can't really provide help with that, sorry. If you find a solution I'm happy to add something to the README.

**Comment by geek876** [comment]
@mikehaertl Thanks. That fixed it. I set the protected `$binPath = '/usr/local/bin/wkhtmltopdf'; `and it started to work. Previously I had 
```
	protected $binPath = '';
	protected $binName = '/usr/local/bin/wkhtmltopdf';
```
I also tested it with
```
	protected $binPath = '/usr/local/bin';
	protected $binName = 'wkhtmltopdf';
```
And With
```
	protected $binPath = '/usr/local/bin/';
	protected $binName = 'wkhtmltopdf';
```
But none of the above 2 settings worked. To be honest they looked more logical whereby the `binPath` is your `bin` directory path and `binName` is the executable name but it did not helped. 
Setting `binName` to something wrong like `wkhtmltopdfxxx` which does'nt exist DOESN'T break it either so I guess that variable is not having any impact.
